src/services/windowTitleService.js

```diff
@@ -29,23 +29,56 @@ const originalWindowTitle = document.title
  * automatically adjusts the title of the window
  *
  * @param {VueRouter} router The VueJS Router instance
+ * @param {Store} store The vuex store
  */
-export default function(router) {
+export default function(router, store) {
+	/**
+	 * Updates the title of the window
+	 *
+	 * @param {Date} date viewed Date
+	 * @param {String} view Name of the current view
+	 * @param {String} locale Locale to be used for formatting
+	 */
+	function updateTitle(date, view, locale) {
+		const title = dateRangeFormat(date, view, locale)
+		document.title = [
+			title,
+			originalWindowTitle,
+		].join(' - ')
```

Comment:
Maybe use translation system (with a comment for context) instead ? Not sure how this work with RTL languages for instance.

Reply:
Hm, I think putting up a translation like `{Date} - {windowtitle}` will cause a lot of confusion for translators. 
Imho it would be better if nextcloud/l10n provided an isRTL method, so we can simply flip it around.